### ISSUE TITLE
Archive?

### ISSUE THREAD

**Issue by story645** [matplotlib/matplotblog#69]
Since the content was moved to https://blog.scientific-python.org/, should this repo be archived, w/ a note on the readme to submit new content there?

**Comment by davidevaleriani** [comment]
Yes, will do. I was waiting for all content to be migrated (for example, this
post <https://matplotlib.org/matplotblog/posts/how-to-create-custom-tables/>
is not there yet)

**Comment by tupui** [comment]
See https://github.com/scientific-python/blog.scientific-python.org/pull/50 😃
